client/my-sites/hosting/sftp-card/index.js

```diff
@@ -92,21 +92,23 @@ const SftpCard = ( {
 		}
 	}, [ username, password ] );
 
-	const renderPasswordCell = () => {
+	const renderPasswordField = () => {
 		if ( disabled ) {
 			return <span></span>;
 		}
 
 		if ( password ) {
 			return (
 				<>
-					<span className="sftp-card__hidden-overflow">{ password }</span>
-					<ClipboardButton text={ password } onCopy={ () => setIsCopied( 'password' ) } compact>
-						{ passwordIsCopied ? translate( 'Copied!' ) : translate( 'Copy password' ) }
-					</ClipboardButton>
+					<div className="sftp-card__copy-field">
+						<FormTextInput className="sftp-card__copy-input" value={ password } onChange={ noop } />
+						<ClipboardButton className="sftp-card__copy-button" text={ password } compact>
+							{ translate( 'Copy' ) }
```

Comment:
Copy has more than 1 meaning, it'd be helpful to provide context for translators